src/baseSwagger.ts

```diff
@@ -153,8 +153,9 @@ export function getSwaggerSchema(paths: IPath, config: IConfig): ISwagger {
 			url: 'http://swagger.io'
 		},
 		components: {
+			...components,
 			securitySchemes: swaggerInitInfo?.security ? getSecurityScheme(swaggerInitInfo.security.methods) : {},
-			schemas: undefined
+			schemas: components.schemas
```

Comment:
@lubomir-dlhy nestacilo len odstranit ten povodny zapis ked vyssie pouzivas object destructuring ?

Reply:
To som nechal z dovodu, ze nesedel potom required typ v interfaci. Upravil som interface a ponechal iba object desctructuring.